scripts/local_environment.sh

```diff
@@ -83,5 +95,26 @@ function main() {
     done
 }
 
+function main() {
+    RUN_COMMAND=deploy
+    parse_arguments "$@"
```

Comment:
The default `RUNTIME` should be set here too

Reply:
The RUNTIME already defaulted at line number 3 alongside with other default vars.

Reply:
Ah yes, sorry, I had only checked the diff.